schemas/concrete/MPTableElementFooter.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$id": "MPTableElementFooter.json",
+  "title": "TableElementFooter",
+  "$mash": {
+    "sources": [{ "$ref": "abstract/MPModel.json#" }
```

Comment:
This should inherit from `ContainedModel` and `ManuscriptModel`, right?

Reply:
yes, will need to include:
```
{ "$ref": "mixins/MPContainedDocument.json#" },
{ "$ref": "mixins/MPManuscriptDocument.json#" },
```

Reply:
Updated.